packages/model-viewer/src/features/loading.ts

```diff
@@ -416,6 +428,12 @@ export const LoadingMixin = <T extends Constructor<ModelViewerElementBase>>(
           this[$posterContainerElement].classList.remove('quick');
         }
       }
+
+      if (changedProperties.has('withCrendentials')) {
+        if (this.withCredentials) {
```

Comment:
I don't think you want this if statement; it would only allow you to set the value to `true`, but not back to `false`.

Reply:
That make sense ! I'm removing the if statement it. 👍